### DISCUSSION TITLE
Prompt Management: Deployment level view

### DISCUSSION THREAD

**Discussion by jannikmaierhoefer** [langfuse/langfuse#4103] [Ideas]
### Describe the feature or potential improvement

Separate view that allowed you to describe the different deployment levels in swim lanes, almost like a Trello board. So you could see the state of each prompt and just drag it to the next stage when ready.

**Comment by marcklingen** [comment]
Sounds cool, let's see how many people are interested in a new view for prompts in Langfuse prompt management to figure out how this might look like.
